### PR TITLE
Service load balancer oper state changes and other minor fixes

### PR DESCRIPTION
**Pull request by abhinandanpb**
The following PR contains:
1) service lb inspect changes
2) service lb fixes
3) Bgp fixes

### CODE REVIEW

File: Godeps/_workspace/src/github.com/contiv/contivmodel/servicelb.json [hunk 1]
```diff
@@ -43,6 +43,22 @@
...
             }
         },
+        "operProperties": {
+          "serviceVip": {
+            "type": "string",
+            "title": "allocated IP addresses"
+          },
+          "numProviders": {
+            "type": "int",
+            "items": "ndpoint",
```
Comment (jainvipin) [inline Godeps/_workspace/src/github.com/contiv/contivmodel/servicelb.json]: `ndpoint`? why do we need to define items for `int` types anyways?

Comment (abhinandanpb) [inline Godeps/_workspace/src/github.com/contiv/contivmodel/servicelb.json]: corrected it.

Comment (abhinandanpb) [inline Godeps/_workspace/src/github.com/contiv/contivmodel/servicelb.json]: I only committed the Godeps here for sanity to pass once. I will be updating godeps and recommiting it.

**Comment by shaleman** [comment]
LGTM

**Pull request merged**